Content.Shared/Implants/Components/ImplanterComponent.cs

```diff
@@ -70,6 +70,12 @@ public sealed partial class ImplanterComponent : Component
     [DataField]
     public (string, string) ImplantData;
 
+    /// <summary>
+    /// If the <see cref="Implant"/> is already implanted, should it be allowed to be implanted again?
+    /// </summary>
+    [DataField]
+    public bool AllowMultipleImplants = true;
```

Comment:
make this false by default. i feel like this this is the exception, not the rule